otherlibs/stdune/src/stdune.mli

```diff
@@ -0,0 +1,145 @@
+[@@@alert unstable "The API of this library is not stable and may change without notice."]
+[@@@alert "-unstable"]
+
+module Appendable_list = Appendable_list
+module Nonempty_list = Nonempty_list
+module Ansi_color = Ansi_color
+module Array = Array
+module Bytes = Bytes
+module Char = Char
+module Comparator = Comparator
+module Either = Either
+module Exn = Exn
+module Exn_with_backtrace = Exn_with_backtrace
+module Filename = Filename
+module Filename_set = Filename_set
+module Hashtbl = Hashtbl
+module Table = Table
+module Int = Int
+module Id = Id
+module Io = Io
+module List = List
+module Map = Map
+module Option = Option
+module Or_exn = Or_exn
+module Ordering = Ordering
```

Comment:
Do we need this mli? It's a bit of a copy-paste that doesn't really define the interface in a more useful way than the .ml file?

Reply:
I think we need it if we want to include the unstable alert with it. Otherwise putting it in the `.ml` file triggers warning 53.

Reply:
Hmm, but why? Are we not allowed to use alerts in .ml files?

Reply:
You can check how they're rejected by 5.2 here: https://github.com/ocaml/ocaml/blob/1f3c26c89b87d9c610699253b605b3fbace2f070/testsuite/tests/warnings/w53.ml#L26

I pushed an alternative version that disables it for the alert attribute in the .ml file